src/models/user-groups-model.js

```diff
@@ -0,0 +1,9 @@
+import database from '../config/database';
+import User from './user-model';
+import Group from './group-model';
+
+const UserGroups = database.define('User_Groups');
```

Comment:
this should be defined after rlationships are added (I mean after `belongsToMany` methods)